src/components/v2/Select/index.tsx

```diff
@@ -0,0 +1,90 @@
+/** @jsxImportSource @emotion/react */
+import React, { useState } from 'react';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import { SelectChangeEvent } from '@mui/material/Select';
+import { Select as MuiSelect } from '@mui/material';
+import Typography from '@mui/material/Typography';
+
+import { Icon } from '../Icon';
+import { TextButton } from '../Button';
+import { SELECTED_MENU_ITEM_CLASSNAME, useStyles } from './styles';
+
+type Option = {
+  value: string;
+  label: string;
+};
+
+interface ISelectProps {
+  className?: string;
+  options: Option[];
+}
+
+const DEFAULT_OPTION = {
+  value: '',
+  label: 'All',
+};
```

Comment:
I'd remove that to make it more flexible (not all selects will have a default "All" option).